platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/api/internal/level/HandlerLoggingLevelConfig.java

```diff
@@ -165,7 +165,7 @@ private Map<String, MarkerState> readMarkers(
         configuration.getPropertyNames().filter(n -> n.startsWith(fullPrefix)).forEach(configPropertyName -> {
             final String name = configPropertyName.substring(fullPrefix.length());
             final MarkerState markerState =
-                    configuration.getValue(configPropertyName, MarkerState.class, MarkerState.UNDEFINED);
+                    configuration.getValue(configPropertyName, MarkerState.class, MarkerState.DEFAULT);
```

Comment:
Open for consideration: for the sake of maintaining a healthy git history, I think it is worth moving this change to its own ticket; if not, this change would appear in the history as "chore: Add first logging documentation," and we would hide the real motive behind the change.